### ISSUE TITLE
We shouldn't rely on CKEDITOR

### ISSUE THREAD

**Issue by adelura** [cksource/samples-framework#35]
In [here](https://github.com/cksource/samples-framework/blob/master/components/tree-a/js/tree-a.js#L10) we rely on CKEDITOR namespave but it won't be available all the time i.e. CKFinder samples.

**Comment by oleq** [comment]
You're right. This should become a vanilla DOM retrieval.

**Comment by adelura** [comment]
I wonder about strategy for old IE`s. Do we want to iclude extra script which will fix issues? For example lack of [classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) object. Do we prefer polyfill or maybe introduce some global object which will implement functionalities which we need?

``` js
var elem = document.getElementById( 'foo' );

// sf stands for samples framework
sf.classList.add( elem, 'bar' );
```

**Comment by Reinmar** [comment]
I don't think we need a polyfill now. There won't be enough JS yet to benefit from it.

**Comment by adelura** [comment]
Thix issue was fixed via #34. I intruduced JavaScript SF object with functionalities which will be described in #37.
